### ISSUE TITLE
Fix for #937 breaks fncall syntax highlight

### ISSUE THREAD

**Issue by dankilman** [pkulchenko/ZeroBraneStudio#957]
Before said commit https://github.com/pkulchenko/ZeroBraneStudio/commit/a700f057248dd17dfceef44ece69c41a1b2d354d
code like this:
```lua
some_module.some_function()
```
would highlight `some_function()` correctly.
After said commit, only if function invocation is preceded with an equals sign (`=`) will it highlight properly, e.g:
```lua
local my_var = some_module.some_function()
```

Function invocations that are not nested (with dots), e.g:
```lua
my_function()
```
are highlighted correctly regardless.

**Comment by pkulchenko** [comment]
I'll check; thank you for the report and for taking time to bisect!

**Comment by dankilman** [comment]
syntax highlight is important to me :)

**Comment by pkulchenko** [comment]
This has been broken by a fix for #937.

**Comment by pkulchenko** [comment]
@dankilman, this issue should be fixed.
